components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/validators/TokenValidationHandler.java

```diff
@@ -711,4 +719,14 @@ private boolean isIDTokenEncrypted(String idToken) {
         return StringUtils.countMatches(idToken, ".") == 4;
     }
 
+    private boolean isOriginatedFromAuthenticationAPI(OAuth2TokenValidationRequestDTO
+                                                              .TokenValidationContextParam[] context) {
+
+        // If the origin of the request is from the authentication API "OAuth.AuthFlow" should be set as
```

Comment:
The comment might be misleading as "OAuth.AuthFlow" has changed to "OAuth.isAuthenticationFlow"

Reply:
Fixed